lib/views/pages/home/content/map/post_map.dart

```diff
@@ -62,24 +64,46 @@ class PostMap extends ConsumerWidget {
       loading: () => (),
     );
 
+    final fab = FloatingActionButton(
```

Comment:
I believe we could rename this to `followButton` or something similar.
Or if you have a better name ?

Reply:
I think it's a good name :)
Fixed in 95818ffa845d1c1e93ba78c50d72a88093cea8a1